### DISCUSSION TITLE
vuejs3 dynamic attribute with no value

### DISCUSSION THREAD

**Discussion by cherimo** [quasarframework/quasar#15284] [General - Components / Directives / etc] [answered]
How can I set up a dynamic attribute within vuejs3. Vanilla js is a lot easier, but within Vue this is apparently not obvious.

I want to be able to use a variable as an attribute with no value at all.

Something like this:

```
  <q-input
     outlined <---(This must be variable "item.design" where it can hold the value outlined, filled etc...)
     v-model="data.value"
     maxlength="12"
     class="super-small subshadow-25"
   />
```
I've read some examples and documentation but the examples are mainly for vuejs2.

Do I miss something?

**Comment by MilosPaunovic** [comment, marked as answer]
You can use `:outlined="item.design"`.
https://vuejs.org/guide/components/props.html#static-vs-dynamic-props

Please try to ask Vue only related questions on a Vue repository, as this has nothing to do with Quasar itself.
